packages/react-icons-mdl2/src/utils/createSvgIcon.ts

```diff
@@ -15,7 +17,18 @@ const createSvgIcon = <TProps = {}>({ svg, displayName }: SvgIconCreateFnParams<
           role: 'presentation',
           ['aria-hidden']: true,
         };
-
+    if (icons?.icons?.[displayName]) {
+      return React.createElement(
+        'span',
+        {
+          ...containerProps,
+          ...nativeProps,
+          className: css(classes.root, className),
+          style,
+        },
+        icons.icons[displayName],
```

Comment:
```suggestion
        icons?.icons?.[displayName] ? icons.icons[displayName] : svg({ classes, props })
```
Here's an example, but the change to be made should actually remove this entire block and edit line 27 of the original file.